BentoKit/BentoKit/Components/TitledDescription.swift

```diff
@@ -508,6 +508,35 @@ private extension Component.TitledDescription.View {
 }
 
 public extension Component.TitledDescription {
+    public final class ContentStyleSheet: ViewStyleSheet<BaseStackView> {
+        public var spacing: CGFloat
+        public var distribution: UIStackView.Distribution
+        public var alignment: UIStackView.Alignment
+        public var isLayoutMarginsRelativeArrangement: Bool
+
+        public init(
+            spacing: CGFloat = 16,
+            distribution: UIStackView.Distribution = .fill,
+            alignment: UIStackView.Alignment = .center,
+            isLayoutMarginsRelativeArrangement: Bool = false
+            ) {
+            self.spacing = spacing
+            self.distribution = distribution
+            self.alignment = alignment
+            self.isLayoutMarginsRelativeArrangement = isLayoutMarginsRelativeArrangement
+        }
+
+        public override func apply(to element: BaseStackView) {
+            super.apply(to: element)
+            element.spacing = spacing
+            element.distribution = distribution
+            element.alignment = alignment
+            element.isLayoutMarginsRelativeArrangement = isLayoutMarginsRelativeArrangement
+            element.cornerRadius = cornerRadius
+            element.borderColor = borderColor?.cgColor
+            element.borderWidth = borderWidth
+        }
+    }
 
     public final class StyleSheet: BaseViewStyleSheet<View> {
```

Comment:
Can you please make it a subclass of interactive view stylesheet as discussed yesterday?